### PR TITLE
Add delete button functionality

### PR DESCRIPTION
**Pull request by Denzel-Turambi**
Changes made in this branch will add delete button functionality, which will delete the idea cards from the DOM and ideas array.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -62,12 +62,17 @@ function addIdeaCard(idea) {
     addIdeaCard(idea);
     clearInputFields();
     toggleButton();
-    console.log("does save btn work?")
 }
   function deleteIdea(event) {
     if(event.target.className === "delete-btn") {
-      event.target.parentNode.parentNode.remove()
```
Comment (asutherland91) [inline main.js:69]: Good idea assigning this to a variable!

**Review by RandyGitProjects — COMMENTED** [review]
Good job on refactoring our function for the deleteIdea.

**Review by brendel-r — COMMENTED** [review]
Great job making the code remove from both the data model and the dom.

**Review by asutherland91 — COMMENTED** [review]
This is looking really good!

**Pull request merged**